Sat Jul 29 14:30:28 PDT 2017 # Analysis Began
Provided cluster nodes: 1000
Job counts:
	count	jobs
	1	bam2bw
	1	bam_finalize
	1	phantom
	1	vanilla
	4

rule bam_finalize:
    input: qc/test.dup.flagstat.qc, bam/test.dupmark.bam
    output: qc/test.filt.nodup.srt.pbc.qc, qc/test.filt.nodup.srt.flagstat.qc, bam/test.filt.nodup.srt.bam
    jobid: 3
    wildcards: sample=test

samtools view -F 1804 -b bam/test.dupmark.bam > bam/test.filt.nodup.srt.bam;samtools index bam/test.filt.nodup.srt.bam;samtools flagstat bam/test.filt.nodup.srt.bam > qc/test.filt.nodup.srt.flagstat.qc;bedtools bamtobed -i bam/test.dupmark.bam |awk 'BEGIN{OFS="\t"}{print $1,$2,$3,$6}' |grep -v 'chrM' | sort -T tmp/test | uniq -c |awk 'BEGIN{mt=0;m0=0;m1=0;m2=0} ($1==1){m1=m1+1} ($1==2){m2=m2+1}{m0=m0+1}{mt=mt+$1} END{printf"%d\t%d\t%d\t%d\t%f\t%f\t%f\n",mt,m0,m1,m2,m0/mt,m1/m0,m1/m2}' > qc/test.filt.nodup.srt.pbc.qc
Waiting at most 5 seconds for missing files.
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/dag.py", line 319, in check_and_touch_output
    wait_for_files(expanded_output, latency_wait=wait)
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/io.py", line 381, in wait_for_files
    latency_wait, "\n".join(get_missing())))
OSError: Missing files after 5 seconds:
qc/test.filt.nodup.srt.pbc.qc
qc/test.filt.nodup.srt.flagstat.qc
bam/test.filt.nodup.srt.bam

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/lib/python3.4/threading.py", line 921, in _bootstrap_inner
    self.run()
  File "/opt/python/lib/python3.4/threading.py", line 869, in run
    self._target(*self._args, **self._kwargs)
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/executors.py", line 568, in _wait_for_jobs
    self.finish_job(active_job.job)
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/executors.py", line 475, in finish_job
    super().finish_job(job, upload_remote=False)
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/executors.py", line 153, in finish_job
    super().finish_job(job, upload_remote=upload_remote, ignore_missing_output=ignore_missing_output)
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/executors.py", line 111, in finish_job
    self.dag.check_and_touch_output(job, wait=self.latency_wait, ignore_missing_output=ignore_missing_output)
  File "/home/shz254/py34env/lib/python3.4/site-packages/snakemake/dag.py", line 323, in check_and_touch_output
    "wait time with --latency-wait.", rule=job.rule)
snakemake.exceptions.MissingOutputException: Missing files after 5 seconds:
qc/test.filt.nodup.srt.pbc.qc
qc/test.filt.nodup.srt.flagstat.qc
bam/test.filt.nodup.srt.bam
This might be due to filesystem latency. If that is the case, consider to increase the wait time with --latency-wait.

Terminating processes on user request.
Will exit after finishing currently running jobs.
Removing output files of failed job bam_finalize since they might be corrupted:
qc/test.filt.nodup.srt.pbc.qc, qc/test.filt.nodup.srt.flagstat.qc, bam/test.filt.nodup.srt.bam
Sat Jul 29 14:32:05 PDT 2017 # Analysis finished
